packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -1001,11 +1012,35 @@ export class FluidDataStoreRuntime
 	}
 
 	public async applyStashedOp(content: any): Promise<unknown> {
-		const envelope = content as IEnvelope;
-		const channelContext = this.contexts.get(envelope.address);
-		assert(!!channelContext, 0x184 /* "There should be a channel context for the op" */);
-		await channelContext.getChannel();
-		return channelContext.applyStashedOp(envelope.contents);
+		const type = content?.type as DataStoreMessageType;
+		switch (type) {
+			case DataStoreMessageType.Attach: {
+				const attachMessage = content.content as IAttachMessage;
+				// local means this node will throw if summarized; this is fine because only interactive clients will have stashed ops
```

Comment:
Based on this comment: https://github.com/microsoft/FluidFramework/blob/bd2c64399366a7173d43f11fab91f21bb6737b88/packages/runtime/runtime-definitions/src/dataStoreContext.ts#L468

I don't really know what this is for.

Reply:
i think it just means the summarizer client will never go down this code path, so its ok, but would be a problem if it did